packages/wrangler/src/config/environment.ts

```diff
@@ -32,8 +32,11 @@ export type Route =
  * Configuration in wrangler for Cloudchamber
  */
 export type CloudchamberConfig = {
+	image?: string;
```

Comment:
how do these new field affect [cloudchamber config validation](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/config/validation.ts#L2207-L2228)?

Reply:
Hmm good call - actually as I read the function, we've got a bunch of code that never gets executed in there to validate value types. Will fix.